NOTES-Perl.md

```diff
@@ -57,18 +57,22 @@
  ourselves to core Perl modules to keep the requirements down.  There
  are just a few exceptions:
 
- Test::More         We require the minimum version to be 0.96, which
-                    appeared in Perl 5.13.4, because that version was
-                    the first to have all the features we're using.
-                    This module is required for testing only!  If you
-                    don't plan on running the tests, you don't need to
-                    bother with this one.
+  * `Test::More`
 
- Text::Template     This module is not part of the core Perl modules.
-                    As a matter of fact, the core Perl modules do not
-                    include any templating module to date.
-                    This module is absolutely needed, configuration
-                    depends on it.
+    We require the minimum version to be 0.96, which
+    appeared in Perl 5.13.4, because that version was
+    the first to have all the features we're using.
+    This module is required for testing only!
+    If you don't plan on running the tests,
+    you don't need to bother with this one.
```

Comment:
> So... Are we implicitly recommending that it is fine to install openssl from sources on a platform in which tests cannot be run?

Good question.
@levitte, @mattcaswell?

Reply:
> So... Are we implicitly recommending that it is fine to install openssl from sources on a platform in which tests cannot be run?

This question is still unanswered...

Reply:
@romen, I just move your question into a separate issue: #12317
since it is actually independent of the re-structuring of the doc files here
in order to prevent this PR from being needlessly blocked.